actions/resources.ts

```
@@ -228,32 +248,31 @@ export async function updateResource(resourceId: string, data: ResourceFormValue
-      return { error: error || 'Unauthorized - Admin access required' }
+      return { error: error || "Unauthorized - Admin access required" };
     }
 
     await prisma.resource.delete({
-      where: { id: resourceId }
-    })
-    return { success: true }
+      where: { id: resourceId },
+    });
+    return { success: true };
   } catch (error) {
```

ESLint

[error] 'error' is defined but never used.

(@typescript-eslint/no-unused-vars)